pydis_site/apps/api/models/bot/nomination.py

```diff
@@ -35,6 +35,11 @@ class Nomination(ModelReprMixin, models.Model):
         default=False,
         help_text="Whether a review was made."
     )
+    thread_id = models.BigIntegerField(
+        help_text="The nomination vote's thread id.",
+        null=True,
+        blank=True
```

Comment:
`blank=True` only makes sense on string fields, as far as I'm aware? https://docs.djangoproject.com/en/4.1/ref/models/fields/#blank

Reply:
You're correct, it's mostly used for validation purposes.

Having it as nullable should suffice. 

I'll remove it